metrics/prometheus/prometheus.go

```diff
@@ -134,21 +134,21 @@ func RegisterCallbackGaugeWithLabels(namespace, subsystem, name, help string, co
 	))
 }
 
-type prometheusHistogram struct {
+type prometheusSummary struct {
 	*prometheus.SummaryVec
 	Pairs map[string]string
 }
 
-// NewHistogram returns a new Histogram backed by a Prometheus summary. It
-// uses a 10-second max age for bucketing. The histogram is automatically
-// registered via prometheus.Register.
-func NewHistogram(namespace, subsystem, name, help string, fieldKeys []string) metrics.Histogram {
-	return NewHistogramWithLabels(namespace, subsystem, name, help, fieldKeys, prometheus.Labels{})
+// NewSummary returns a new Histogram backed by a Prometheus summary. It uses
+// a 10-second max age for bucketing, emulating statsd. The histogram is
```

Comment:
Not related to this PR, but:

10s is probably fine for high-qps environment, but is it really intended to disregard any observation older than 10s, no matter what? (This is not configurable in gokit, as I understand...) Note that the default is 10m, two orders of magnitude more...